### PR TITLE
Switch subgraph URLs to decentralized network

### PR DESCRIPTION
**Pull request by 0xAurelius**
DO NOT MERGE UNTIL ALL SUBGRAPHS ARE UPGRADED!

### CODE REVIEW

File: flows/constants.py [hunk 1]
```diff
@@ -1,21 +1,48 @@
...
-CARBON_HOLDERS_SUBGRAPH_URL = (
-    "https://api.thegraph.com/subgraphs/name/klimadao/klimadao-user-carbon"
-)
-PAIRS_SUBGRAPH_URL = (
-    "https://api.thegraph.com/subgraphs/name/klimadao/klimadao-pairs"
-)
+import os
+
+GRAPH_API_KEY = os.environ.get('GRAPH_API_KEY')
+GRAPH_BASE_URL = f'https://gateway-arbitrum.network.thegraph.com/api/{GRAPH_API_KEY}/subgraphs/id/'
```
Comment (biwano) [inline flows/constants.py:4]: Should you not be using this in the `production` configuration?

Comment (0xAurelius) [reply flows/constants.py:4]: yes we will, this is still a draft

File: flows/constants.py [hunk 2]
```diff
@@ -1,21 +1,44 @@
...
+GRAPH_VERSION_SUFFIX = '/version/latest'
+
+CARBON_SG_ID = '5WCNJmuBbejEU3jqL5Hswde5Vkn2yrQWfeJrhUSRGp8J'
+CARBON_LEGACY_SG_ID = '2z66bx6cRCMHrkBcd5ZhN2Z6JgCtntgHFemgtte3qr82'
+CARBON_ETH_SG_ID = '4krJ9sutnjfZgKUjfKcnnukRqwsAycfXKvdApHbKf8zf'
+CARBON_CELO_SG_ID = 'Cv1ZbYYcWp9oKEdaygT8ui9yzgMLLJjdhWeVRmD6yTfA'
+CARBON_HOLDERS_SG_ID = 'DAd5YXDcVuHtYYAcqViqWDqE1hSRCrVULvcBWQHojCxD'
+PAIRS_SG_ID = 'Fd2ydz1RopNrgtH9eoHeVrsCPjg7ACWxyLKJKSVeHjW6'
+
+if os.environ.get('ENV') == 'production':
```
Comment (biwano) [inline flows/constants.py:15]: Is that environment variable set?

**Comment by 0xAurelius** [comment]
Yes it should be, I added it to the k8s setup elsewhere in this PR

-------- Original Message --------

**Review by biwano — APPROVED** [review]

**Pull request merged**